src/pusb.c

```diff
@@ -506,69 +498,32 @@ int main(int argc, char *argv[])
 		help();
 		return -2;
 	}
-
+/*
 	if (verbose == 2) {
 		usb_set_debug(255);
 	}
-	usb_init();
-	usb_find_busses();
-	old_dev = 0;
+	*/
+	libusb_init(NULL);
+	//usb_find_busses();
 	N_PRINT("Waiting for USB device vendorID=0x%X "
 		"and productID=0x%X:\n", search_vendor, search_product);
-	while (!found) {
+	while (1) {
 		/* Enumerate buses once more.. just in case
 		 * we have some one plugging in a device with a hub..
 		 */
-		new_dev = usb_find_devices();
-		if (new_dev == old_dev) {
-			/* no change in bus state.. BRUTE retry */
-			usb_sleep(10);
-			continue;
-		}
-		old_dev = new_dev;
-		/* new device detected */
-		for (bus = usb_get_busses(); bus; bus = bus->next) {
-			if (bus->root_dev)
-				found = search_device(bus->root_dev, 0);
-			else {
-				struct usb_device *dev;
-
-				for (dev = bus->devices; dev; dev = dev->next) {
-					found = search_device(dev, 0);
-					if (found)
-						break;
-				}
-			}
-			if (found)
-				break;
-		}
-		V_PRINT("Bus Scan %d:\n", x++);
-
+                udev = libusb_open_device_with_vid_pid(NULL,  search_vendor, search_product);
+		if (udev)
+		  break;
+                usb_sleep(200);
```

Comment:
why?

Reply:
why what? The usb_sleep()? I did not want to fully busy-waiting here().
Anything else?

Reply:
@akemnade  duration - why 200 ?